cmd/completion.go

```diff
@@ -20,10 +20,17 @@
  *
  */
 
-package main
+package cmd
 
-import "github.com/skydive-project/skydive/cmd"
+import "github.com/spf13/cobra"
 
-func main() {
-	cmd.RootCmd.GenBashCompletionFile("skydive-bash-completion.sh")
+// BashCompletion skydive root command
+var BashCompletion = &cobra.Command{
+	Use:          "bash-completion",
+	Short:        "generate skydive-bash-completion.sh bash helper",
+	Long:         "generate skydive-bash-completion.sh bash helper",
+	SilenceUsage: true,
+	PersistentPreRun: func(cmd *cobra.Command, args []string) {
```

Comment:
It does nothing on my machine. You probably need to change PersistentPreRun to Run